### PR TITLE
Feat/friend page

### LINKED ISSUE #13: localStorage for Friendlist
# Value Statement
As a user of the App, I want to be able to see my friends which I have stored in localStorage render in the friend list so that I can have _more friends_.

# Description

Screenshot 2021-04-28 at 10 34 27

Screenshot 2021-04-28 at 10 37 01

# Acceptance Criteria

- The friend page has a "add new friend" section.
- There will be a add friend form consisting of `name` and `image`.

# Tasks
- [x] create an `add friend` icon.
- [x] create a `form`. 
- [x] The form will pop up on the screen.
- [x] inputs from the form will be sent into localStorage and be displayed in the friend page. 

# Complexity

Medium/High

### PR DESCRIPTION
**Pull request by hugh-burgess**
resolves #13

### CODE REVIEW

File: client/src/components/friends-page/AddButton.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  function handleNameChange(event) {
+    setFriendName(event.target.value);
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    // const formData = new FormData();
+    // formData.append("name", friendName);
+    // formData.append("file", selectedFile);
```
Comment (FriederikeM) [inline client/src/components/friends-page/AddButton.js:31]: Commented code? no thanks!

File: client/src/components/friends-page/AddButton.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+    addItemToLocalStorage({
+      name: friendName,
+      // file: selectedFile,
```
Comment (FriederikeM) [inline client/src/components/friends-page/AddButton.js:36]: no no

File: client/src/components/friends-page/AddButton.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  //   formData.append("file", selectedFile);
+
+  //   axios
+  //     .post(url, formData)
+  //     .then((res) => {
+  //       alert("File Upload success");
+  //       console.log("File Upload success");
+  //     })
+  //     .catch((err) => alert("File Upload Error"));
+  // };
```
Comment (FriederikeM) [inline client/src/components/friends-page/AddButton.js:73]: Don't merge them comments

File: client/src/components/friends-page/AddButton.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+  function handleButtonClick(event) {
+    event.preventDefault();
+    setIsClicked(!isClicked);
+    console.log(isClicked);
```
Comment (FriederikeM) [inline client/src/components/friends-page/AddButton.js:78]: you can delete the console log

File: client/src/components/friends-page/AddButton.js [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+              type="text"
+              name="friendName"
+              placeholder="name goes here..."
+              onChange={handleNameChange}
+              value={friendName}
+            />
+            {/* <FileUploader
+              onFileSelectSuccess={(file) => setSelectedFile(file)}
+              onFileSelectError={({ error }) => alert(error)}
+            /> */}
```
Comment (FriederikeM) [inline client/src/components/friends-page/AddButton.js]: more

**Review by FriederikeM — CHANGES_REQUESTED** [review]
LGTM, but there is a lot of commented code that you want to get rid of before you merge

**Review by soerendolling — APPROVED** [review]
Looks good to me

**Pull request merged**